src/test/kotlin/no/nav/dagpenger/inntekt/ApiTest.kt

```diff
@@ -0,0 +1,23 @@
+package no.nav.dagpenger.inntekt
+
+import com.google.gson.Gson
+import io.ktor.application.Application
+import io.ktor.http.HttpMethod
+import io.ktor.http.HttpStatusCode
+import io.ktor.server.testing.handleRequest
+import io.ktor.server.testing.withTestApplication
+import java.math.BigDecimal
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+class ApiTest {
+
+    @Test fun ` should be able to Http GET inntekt `() = withTestApplication(Application::main) {
```

Comment:
ktor-server-test-host ^ library.  See https://ktor.io/servers/testing.html#testengine 
A this do not start any real http server it is superfast

Reply:
Cooolio!